pom.xml

```diff
@@ -67,7 +68,7 @@
     <module>dynamodb</module>
     <module>elasticsearch</module>
     <!--<module>gemfire</module>-->
-    <module>infinispan</module>
```

Comment:
please don't include changes to other modules. if infinispan currently isn't building, we can disable it as a different PR.